src/app.js

```diff
@@ -0,0 +1,72 @@
+const p = require('path');
```

Comment:
We won't be needing this file

Reply:
We need this path module because in index.js file when we are reading the templates it is used the resolve the correct path.
`      const path = p.join(__dirname, './certification/' + v4() + ".pdf")
      const templatePath = p.join(__dirname, './templates/' + data.template + '.hbs')`

Reply:
I was talking about the whole file

Reply:
ok I will do this